ramls/users.raml

```diff
@@ -10,6 +10,8 @@ documentation:
 types:
   user: !include userdata.json
   userdataCollection: !include userdataCollection.json
+  outboxEventLog: !include outboxEventLog.json
+  userConsortiaEvent: !include userConsortiaEvent.json
```

Comment:
Why are these in the RAML definitions when there is no endpoints associated with either of these types?

Reply:
If you don't specify a JSON schema in the RAML definitions, it won't be generated. I thought that user.raml is the most appropriate place.

Reply:
Ok, this is a workaround to generate classes for the event log and user event types?

Reply:
You're partially correct. However, outboxEventLog is an entity for storing payloads for the transactional outbox pattern for any types of events, while userEvent is the payload that will be sent to Kafka. I created a separate enum as a Java class without generating it for event types.

Reply:
> However, outboxEventLog is an entity for storing payloads for the transactional outbox pattern for any types of events, while userEvent is the payload that will be sent to Kafka. 

How does that distinction affect the code generation?

Reply:
we can't reference the userEvent directly in the outboxEventLog fields, we store the userEvent in a json representation in the payload field of the outboxEvenLog. This is done to reuse outboxEventLog for other entities.

Reply:
Sure, I understand that. I don't understand how that affects the code generation. I don't think it's important, I now understand the desire for the code generation and this workaround was considered the best way.